packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -176,6 +176,11 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
    */
   calloutProps?: ICalloutProps;
 
+  /**
+   * the total size of the set
```

Comment:
Can you add more details on when this property should be set and provide guidelines for the value?